kalite/control_panel/templates/control_panel/partials/_facility_table.html

```diff
@@ -27,7 +39,7 @@
                     </thead>
                     <tbody>
                         {% for id,facility in facilities.items %}
-                            <tr>
+                            <tr {% if facility.meta_data_in_need %} class="warning" {% endif %}>
```

Comment:
Maybe we could also add a warning glyphicon next to the facility name in the row too? Might tie nicely together with the warning glyphicon you have at the top of the table. Can make sure it is part of the 'edit facility' link, so that clicking on it would take people directly to the place where they could fill in the info.